store/tikv/lock_resolver.go

```diff
@@ -486,6 +487,7 @@ func (lr *LockResolver) getTxnStatus(bo *Backoffer, txnID uint64, primary []byte
 			status.ttl = cmdResp.LockTtl
```

Comment:
Should also set `status.action` here.

Reply:
Done

Reply:
It's not changed. 😂

Reply:
It's changed...what's wrong?  @youjiali1995 

https://github.com/pingcap/tidb/pull/13432/commits/260d1557170248dfae160ee604bea386445c9df5#diff-39b30a313dde4d126e56f1828b6dabdcR486

Reply:
My fault.😅